config/webpack.common.js

```diff
@@ -126,23 +130,26 @@ module.exports = (env) => {
     },
     output: {
       filename: '[name].bundle.js',
-      path: path.resolve(__dirname, 'dist'),
+      path: path.resolve(__dirname, '../dist'),
     },
     plugins: [
       new HtmlWebpackPlugin(
         env === 'development' || process.env.DATA_SOURCE === 'mock'
           ? {
               // In dev and mock-prod modes, populate window._env at build time
               filename: 'index.html',
-              template: path.resolve(__dirname, 'src', 'index.html.ejs'),
+              template: path.resolve(__dirname, '../src/index.html.ejs'),
               templateParameters: {
-                _env_encoded: require('./runtime-env-vars'),
+                _env_encoded: require('../runtime-env-vars'),
```

Comment:
Let's remove `_env_encoded` entirely and delete the `runtime-env-vars` script. We should also remove everything from `.env.example` except for `DATA_SOURCE` and change the README so it describes adding those variables to virtMeta instead of .env